llvm/test/CodeGen/DirectX/legalize-lifetimes-valver-1.5.ll

```diff
@@ -16,3 +16,6 @@ define void @test_legal_lifetime()  {
   call void @llvm.lifetime.end.p0(i64 4, ptr nonnull %accum.i.flat)
   ret void
 }
+
+!dx.valver = !{!0}
+!0 = !{i32 1, i32 5}
```

Comment:
Can you add a comment here so its clear this is how we are specifying dxil version.